docs/customizing_page_views.md

```diff
@@ -94,3 +94,25 @@ rails generate administrate:views:layout
  # It only generates the sidebar partial
  # -> app/views/admin/application/_navigation.html.erb
 ```
+
+## Adding custom CSS and JS
+
+You can add custom CSS and JS to Administrate. Put the files in the
+appropriate folders (typically under `assets`) and point Administrate to them
+using the following API, preferably in an initializer. For example, if your
+files are called `admin.css` and `admin.js`:
+
+```
+/// config/initializers/administrate.rb
+Administrate::Engine.add_stylesheet("admin")
+Administrate::Engine.add_javascript("admin")
+```
+
+Then make sure to list them in your manifest file (Rails will helpfully remind
+you of this step if you miss it):
+
+```
+/// app/assets/config/manifest.js
+//= link admin.css
+//= link admin.js
+```
```

Comment:
Looks good, thank you!

@pablobm @nickcharlton I wonder what we should to do with the following generators:
- `lib/generators/administrate/assets/assets_generator.rb`
- `lib/generators/administrate/assets/javascripts_generator.rb`
- `lib/generators/administrate/assets/stylesheets_generator.rb`

They are another way of customizing assets. As far as I understand, by running those generators, Administrate's non-compiled assets are copied to the users' app. So users can edit a non-compiled asset and _compile it themselves_ if their app is set up for that. And then their assets will take precedence over the ones from the Administrate gem.

I lean toward removing those generators and making users customize assets by appending to Administrate's CSS/JS, just like the example in this PR, instead of completely replacing them with their own.

Reply:
I agree. I think that this way of customising assets is more likely to cause issues than to solve them. Adding additional, separate assets to add new behaviour (or override CSS) feels like a better solution to me. Then if anyone wants to fully override Administrate's own assets, they can copy them over manually and do that.

Reply:
Yeah, that makes sense to me 👍